nixos/modules/services/databases/riak-cs.nix

```diff
@@ -0,0 +1,201 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.riak-cs;
+
+in
+
+{
+
+  ###### interface
+
+  options = {
+
+    services.riak-cs = {
+
+      enable = mkEnableOption "riak-cs";
+
+      package = mkOption {
+        type = types.package;
+        defaultText = "pkgs.riak-cs";
+        example = literalExample "pkgs.riak-cs";
```

Comment:
Let's just include this anyways: it might come into use in some part of the NixOS system where all examples are iterated over without defaultText, or vice-versa